apps/communications/contacts/test/marionette/lib/views/contact-details/view.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+
+/* global module */
+
+var ContactsDetailsAccessor = require('./accessors');
+
+function ContactsDetailsView(client, subject) {
+	this.client = client;
+	this.subject = subject;
+	this.accessors = new ContactsDetailsAccessor(client);
+}
+
+ContactsDetailsView.prototype = {
+	makeFavorited: function() {
+		this.accessors.favoritesButton.tap();
+		return this.waitForFavorited;
+	},
+
+	waitForFavorited: function() {
+		function waitFav(temp) {
+        	return temp.detailsHeader.getAttribute('class'
+        	).indexOf('favorite') != -1;
+      	}
+      	this.client.waitFor(waitFav(this.accessors));
```

Comment:
:warning: This line should have a 2-space-indentation as it doesn't belong to `waitFav`.